rlp/src/impls.rs

```diff
@@ -79,6 +79,44 @@ impl Decodable for Vec<u8> {
 	}
 }
 
+#[cfg(feature = "arrayvec")]
```

Comment:
I think you missed adding this feature to `Cargo.toml`.

Reply:
Optional dependencies are features in themselves.

Reply:
TIL! 👍 

(I'd still want all features in the `Cargo.toml`, preferably documented, for discoverability reasons)